server/api/routes/meeting.cjs

```diff
@@ -93,6 +94,81 @@ router.delete('/:meetingId', requireToken, isAdmin, async (req, res, next) => {
     next(err);
   }
 });
+router.post('/active/messages', requireToken, async (req, res, next) => {
```

Comment:
Add the sameUserOrAdmin middleware so that anyone with a token can't send a message to anyone